### ISSUE TITLE
Sync UI fixes

### ISSUE THREAD

**Issue by ryanfeeley** [mozilla-mobile/fenix#3772]
A bit disappointed to see what landed for Sync after the big marketing push, but anyway, here's what was supposed to ship before we made waves about it.

![image](https://user-images.githubusercontent.com/68213/60364682-d3516280-99b4-11e9-9bea-0e083704ca84.png)

**Comment by grigoryk** [comment]
@ryanfeeley we ran out of time before the strings/code freeze to implement the new login/pairing UI. These changes should land sometime in the next few sprints.

**Comment by mheubusch** [comment]
@lime124 let's review the request with Ryan and then schedule a design handoff

**Comment by ryanfeeley** [comment]
I look forward to the next release, as some small fixes will make a big difference in the experience.

The only true *bug* I see is that the **Create an account** link should have been removed when we added **Turn on Sync**. They link to the exact same screen. For the quickest fix, remove it ASAP. Longer term, this screen disappears entirely in favour of the page with the illustration.

![image](https://user-images.githubusercontent.com/68213/60384423-781e7f00-9a4b-11e9-8b5c-24cdc338e346.png)

**Comment by mheubusch** [comment]
@vesta0 AFAICT there are no strings needed, just some removed.  I already approved the removal of Already have an account in another issue. Tif and Ryan can work together for a design handoff.

**Comment by ekager** [comment]
@ryanfeeley Let me know when the designs are finalized and I can get an invision link or mock so I can match sizing. I have most of the changes done. Sorry this didn't make it into release!

@mheubusch I'm seeing maybe 2 new strings in the mocks. A new instructions string for pairing and a new title for pairing "Scan the code"

**Comment by ekager** [comment]
Also @ryanfeeley is the dialog in https://github.com/mozilla-mobile/fenix/issues/3134 part of this ticket?

**Comment by vladikoff** [comment]
> Also @ryanfeeley is the dialog in #3134 part of this ticket?

I guess it could be a separate fix?

**Comment by vladikoff** [comment]
(side note, once we land the new screens, we can keep an eye on this chart: https://sql.telemetry.mozilla.org/queries/63433/source#162424  to see how it improves the numbers for pairing)

**Comment by ryanfeeley** [comment]
For now, please remove the redundant bottom link *Create an account*. That's the only part of the descoping that was done without surgical precision. ;)

**Comment by vladikoff** [comment]
thanks Ryan!

**Comment by ekager** [comment]
For now, I've removed the create an account option in #3928 . I've also opened a draft PR with most of the other changes (and strings) https://github.com/mozilla-mobile/fenix/pull/3933 and can update with exact values when ready.

**Comment by vladikoff** [comment]
> I've also opened a draft PR with most of the other changes

<img src=https://user-images.githubusercontent.com/128755/60977009-b1cd7080-a2fc-11e9-9a60-df68a1ac1a0c.png width=500 /> 

* new info from @ryanfeeley, header should say "Turn on Sync" instead of "Sign In"
* the graphic looks squashed on the Essential PH-1, see screenshot
* the instructions are gone from the scanner view. @ryanfeeley 's design has the new design for the scanner. should we leave the instructions for now, until we get the new scanner view?
* Nexus 5 is crashing on pressing "Turn on Sync" with https://gist.github.com/vladikoff/145d0b49917839ac8b86cc98b5c36c4c

@ryanfeeley and I will file more bugs after we land these set of fixes. The title of the issue is a bit generic right now, sorry about that

**Comment by vladikoff** [comment]
@ryanfeeley whats the width and height of the art ? could you please check in Sketch?

**Comment by ryanfeeley** [comment]
320px × 166px

**Comment by ekager** [comment]
>     * the instructions are gone from the scanner view. @ryanfeeley 's design has the new design for the scanner. should we leave the instructions for now, until we get the new scanner view?

**Comment by ekager** [comment]
Okay I landed these first changes! QA will close this bug when verified it does not regress any behavior and works as expected on variety of devices. Please file followup issues for UX polish

**Comment by sv-ohorvath** [comment]
Verified these changes:
- "Already have an account?" removed from the onboarding screen
- Turn on sync screen replaced with new UI
- Pairing instructions dialog removed
- "Scan the code" title replacement

Remaining change not implemented:
The instructions banner on the scanner screen - should be replaced with the one in the description. - future work.

Created new issues for other UI problems found:
The crash on Android 5&6 was logged here: https://github.com/mozilla-mobile/fenix/issues/3997
https://github.com/mozilla-mobile/fenix/issues/3998
https://github.com/mozilla-mobile/fenix/issues/3999
https://github.com/mozilla-mobile/fenix/issues/4001

**Comment by ekager** [comment]
I should have addressed all of the follow  up issues except #3999 , so let's close this issue and that issue can track that work :)
